subsys/nrf_security/src/drivers/Kconfig.psa_accel

```diff
@@ -0,0 +1,821 @@
+	bool
+
+config PSA_ACCEL_ECDSA_P384_SHA512
+	bool
+
+config PSA_ACCEL_ECJPAKE_P256_SHA1
+	bool
+
+config PSA_ACCEL_ECJPAKE_P256_SHA224
+	bool
+
+config PSA_ACCEL_ECJPAKE_P256_SHA256
+	bool
+
+config PSA_ACCEL_ECJPAKE_P256_SHA384
+	bool
+
+config PSA_ACCEL_ECJPAKE_P256_SHA512
+	bool
+
+config PSA_ACCEL_ECJPAKE_TO_PMS
+	bool
+
+config PSA_ACCEL_ED25519PH
+	bool
+
+config PSA_ACCEL_ED448PH
+	bool
+
+config PSA_ACCEL_ENTROPY
+	bool
+
+config PSA_ACCEL_FFDH_2048
+	bool
+
+config PSA_ACCEL_FFDH_3072
+	bool
+
+config PSA_ACCEL_FFDH_4096
+	bool
+
+config PSA_ACCEL_FFDH_6144
+	bool
+
+config PSA_ACCEL_FFDH_8192
+	bool
+
+config PSA_ACCEL_HKDF_EXPAND_SHA1
+	bool
+
+config PSA_ACCEL_HKDF_EXPAND_SHA224
+	bool
+
+config PSA_ACCEL_HKDF_EXPAND_SHA256
+	bool
+
+config PSA_ACCEL_HKDF_EXPAND_SHA384
+	bool
+
+config PSA_ACCEL_HKDF_EXPAND_SHA512
+	bool
+
+config PSA_ACCEL_HKDF_EXTRACT_SHA1
+	bool
+
+config PSA_ACCEL_HKDF_EXTRACT_SHA224
+	bool
+
+config PSA_ACCEL_HKDF_EXTRACT_SHA256
+	bool
+
+config PSA_ACCEL_HKDF_EXTRACT_SHA384
+	bool
+
+config PSA_ACCEL_HKDF_EXTRACT_SHA512
+	bool
+
+config PSA_ACCEL_HKDF_SHA1
+	bool
+
+config PSA_ACCEL_HKDF_SHA224
+	bool
+
+config PSA_ACCEL_HKDF_SHA256
+	bool
+
+config PSA_ACCEL_HKDF_SHA384
+	bool
+
+config PSA_ACCEL_HKDF_SHA512
+	bool
+
+config PSA_ACCEL_HMAC_SHA1
+	bool
+
+config PSA_ACCEL_HMAC_SHA224
+	bool
+
+config PSA_ACCEL_HMAC_SHA256
+	bool
+
+config PSA_ACCEL_HMAC_SHA384
+	bool
+
+config PSA_ACCEL_HMAC_SHA512
+	bool
+
+config PSA_ACCEL_KEY_PAIR_ED25519
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K163
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K192
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K224
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K233
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K239
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K256
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K283
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K409
+	bool
+
+config PSA_ACCEL_KEY_PAIR_K571
+	bool
+
+config PSA_ACCEL_KEY_PAIR_P224
+	bool
+
+config PSA_ACCEL_KEY_PAIR_P256
+	bool
+
+config PSA_ACCEL_KEY_PAIR_P384
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB160
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB192
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB224
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB256
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB320
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB384
+	bool
+
+config PSA_ACCEL_KEY_PAIR_PB512
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R163
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R192
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R233
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R283
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R409
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R521
+	bool
+
+config PSA_ACCEL_KEY_PAIR_R571
+	bool
+
+config PSA_ACCEL_KEY_PAIR_X25519
+	bool
+
+config PSA_ACCEL_MD5
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA1
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA224
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA256
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA384
+	bool
+
+config PSA_ACCEL_PBKDF2_HMAC_SHA512
+	bool
+
+config PSA_ACCEL_RANDOM
+	bool
+
+config PSA_ACCEL_RIPEMD160
+	bool
+
+config PSA_ACCEL_RSA1024_PKCS1V15_CRYPT
```

Comment:
This doesn't match with patterns established for ECDH and ECDSA. Please use the root algorithm after PSA_ACCEL and move key size (and hash algorithms) to the end

https://armmbed.github.io/mbed-crypto/html/api/ops/pke.html?highlight=alg_rsa#c.PSA_ALG_RSA_PKCS1V15_CRYPT

PSA_ACCEL_RSA_PKCS1V15_CRYPT_RSA1024

Reply:
Same as above, these come from the Oberon PSA core.